docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>What do the colors in the dendrogram mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Reading the docs, I am not exactly quite sure what it means, seems like they color it something depending on whether the distance of the child nodes exceeds a certain threshold (default is set to be 70% of max distance). This doesn't seem particularly relevant for this case so I can just set `link_color_func` to be a function that returns the same color for every cluster.